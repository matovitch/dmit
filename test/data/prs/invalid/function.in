func f() {}
func g(i :) {}
func h() {}
