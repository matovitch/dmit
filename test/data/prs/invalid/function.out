{"tree" :{"nodes":[{"kind":"END_OF_TREE","size":0,"childCount":0,"start":0,"stop":0},{"kind":"IDENTIFIER","size":0,"childCount":0,"start":28,"stop":27},{"kind":"FUN_ARGUMENTS","size":0,"childCount":0,"start":27,"stop":24},{"kind":"SCOPE","size":0,"childCount":0,"start":24,"stop":21},{"kind":"DEF_FUNCTION","size":3,"childCount":3,"start":30,"stop":21},{"kind":"DEFINITION","size":4,"childCount":1,"start":30,"stop":21},{"kind":"DEFINITION","size":0,"childCount":0,"start":21,"stop":9},{"kind":"IDENTIFIER","size":0,"childCount":0,"start":7,"stop":6},{"kind":"FUN_ARGUMENTS","size":0,"childCount":0,"start":6,"stop":3},{"kind":"SCOPE","size":0,"childCount":0,"start":3,"stop":0},{"kind":"DEF_FUNCTION","size":3,"childCount":3,"start":9,"stop":0},{"kind":"DEFINITION","size":4,"childCount":1,"start":9,"stop":0},{"kind":"MODULE","size":11,"childCount":3,"start":30,"stop":0}]},"errors":[{"offset":14,"errors":[{"expect":"IDENTIFIER","actual":"PAR_RIGHT","treeNodeKind":"IDENTIFIER"}]},{"offset":4294967295,"errors":[]}]}